tests/e2e/support/objects/app-files/search/actions.ts

```diff
@@ -0,0 +1,44 @@
+import { Page } from 'playwright'
+import util from 'util'
+
+const globalSearchInputSelector = '.oc-search-input'
+const searchResultMessageSelector = '//p[@class="oc-text-muted"]'
+const selectTagDropdownSelector =
+  '//div[contains(@class,"files-search-filter-tags")]//button[contains(@class,"oc-filter-chip-button")]'
+const tagFilterChipSelector = '//button[contains(@data-test-value,"%s")]'
+const clearTagFilterSelector =
+  '//div[contains(@class,"files-search-filter-tags")]//button[contains(@class,"oc-filter-chip-clear")]'
+const enableSearchInFileContentSelector =
+  '//div[contains(@class,"files-search-filter-full-text")]//button[contains(@class,"oc-filter-chip-button")]'
+const disableSearchInFileContentSelector =
+  '//div[contains(@class,"files-search-filter-full-text")]//button[contains(@class,"oc-filter-chip-clear")]'
+
+export interface fullTextSearchArgs {
+  keyword: string
+  page: Page
+}
+
+export const fullTextSearch = async (args: fullTextSearchArgs): Promise<void> => {
```

Comment:
```suggestion
export const searchFullText = async (args: fullTextSearchArgs): Promise<void> => {
```

Reply:
I prefer using `fullTextSearch` because it points towards the feature whereas `searchFullText` might mislead. Also, the `fullTextSearch` term is used in feature implementation too.